subsys/bluetooth/controller/ll_sw/ull_central_iso.c

```diff
@@ -77,6 +80,59 @@ static struct {
 	struct ll_conn_iso_stream stream[CONFIG_BT_CTLR_CONN_ISO_STREAMS_PER_GROUP];
 } ll_iso_setup;
 
+static uint8_t ll_cig_parameters_validate(void)
+{
+	if (ll_iso_setup.cis_count > BT_HCI_CIS_COUNT_MAX) {
+		/* Invalid CIS_Count */
+		return BT_HCI_ERR_INVALID_PARAM;
+	}
+
+	if (ll_iso_setup.group.cig_id > BT_HCI_BT_HCI_CIG_ID_MAX) {
+		/* Invalid CIG_ID */
+		return BT_HCI_ERR_INVALID_PARAM;
+	}
+
+	if (!IN_RANGE(ll_iso_setup.group.c_sdu_interval, BT_HCI_SDU_INTERVAL_MIN,
+		      BT_HCI_SDU_INTERVAL_MAX) ||
+	    !IN_RANGE(ll_iso_setup.group.p_sdu_interval, BT_HCI_SDU_INTERVAL_MIN,
+		      BT_HCI_SDU_INTERVAL_MAX)) {
+		/* Parameter out of range */
+		return BT_HCI_ERR_INVALID_PARAM;
+	}
+
+	if (ll_iso_setup.group.central.test) {
+		if (!IN_RANGE(ll_iso_setup.group.iso_interval,
+			      BT_HCI_ISO_INTERVAL_MIN, BT_HCI_ISO_INTERVAL_MAX)) {
+			/* Parameter out of range */
+			return BT_HCI_ERR_INVALID_PARAM;
+		}
+	} else {
+		if (!IN_RANGE(ll_iso_setup.group.c_latency, BT_HCI_MAX_TRANSPORT_LATENCY_MIN * 1000,
+			      BT_HCI_MAX_TRANSPORT_LATENCY_MAX * 1000) ||
+		    !IN_RANGE(ll_iso_setup.group.p_latency, BT_HCI_MAX_TRANSPORT_LATENCY_MIN * 1000,
+			      BT_HCI_MAX_TRANSPORT_LATENCY_MAX * 1000)) {
```

Comment:
I see that we store the latency in microseconds with 
```c
ll_iso_setup.group.c_latency = c_latency * 1000;
ll_iso_setup.group.p_latency = p_latency * 1000;
```

What is the rationale behind using microseconds?

Also suggest to use `* USEC_PER_MSEC` instead of `* 1000`

Reply:
The rationale was to avoid doing the calculation when using the parameter, as they are referenced multiple times. Not sure changing that is in scope for this PR, unless you insist?

Reply:
> Not sure changing that is in scope for this PR, unless you insist?

No, it's fine to keep as microseconds, but do please update to use `USEC_PER_MSEC` instead of `1000` :)